rpm/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I thought it would be nice to provide such file as a 'learn by example' option. Just removed it. :)